docs/slides/multi-cloud/adp/index.md

```diff
@@ -0,0 +1,634 @@
+name: field-workshops-assets-index
+class: title, shelf, no-footer, fullbleed
+background-image: url(images/HashiCorp-Title-bkg.jpeg)
+<div class="clearfix">
+  <div class="img-container">
+    <img src="images\Vault_VerticalLogo_ColorWhite_RGB.png" style="width:5%">
+  </div>
+  <div class="img-container">
+    <img src="images\Consul_VerticalLogo_ColorWhite_RGB.png" style="width:5%">
+  </div>
+  <div class="img-container">
+    <img src="images\Boundary_VerticalLogo_ColorWhite_RGB.png" style="width:5%">
+  </div>
+</div>
+
+# Zero Trust Security & Data Protection
+## Trust Nothing. Authenticate and Authorize Everything.
+
+???
+
+---
+layout: true
+
+.footer[
+- Copyright © 2021 HashiCorp
+- ![:scale 100%](images/HashiCorp_Icon_Black.svg)
+]
+
+---
+name: intros
+# Introductions
+
+* Your Name
+* Job Title
+* Secrets Management Experience
+
+???
+
+---
+name: prerequisites
+# Prerequisites
+
+- Some very basic familiarity with the following Vault concepts
+  - Interacting with Vault via one of the CLI, GUI, or API
+      - The Vault binary
+      - Client-Server model
+  - Vault Authentication Methods (High-level understanding)
+  - Vault Secret Engines (High-level understanding)
+
+???
+
+---
+name: table of contents
+# Table of Contents
+* HashiCorp Vault Overview
+* Dynamic Database Secrets
+* API Encryption as a Service (& Tokenization)
+* Filesystem/Database/Hardware Encryption as a Service
+
+???
+
+---
+name: environment
+# Lab Environment Used
+* This workshop uses Instruqt for hand-on labs.
+* Instruqt labs are run in “tracks” that are divided into “challenges”
+* This workshop uses the following tracks
+  1. Vault Basics Review
+  1. Vault Dynamic Database Credentials
+  1. Vault Transform and Transit Secrets Engines (Appl Level Encryption)
+  1. (Optional) Vault Transform-Tokenization Secret Engine 
```

Comment:
Change "Secret Engine" and any other occurences to "Secrets Engine" for consistency with Vault doc usage.